packages/core/ios/RNEmbrace/EmbraceManager.swift

```diff
@@ -204,23 +204,22 @@ class EmbraceManager: NSObject {
         _ resolve: @escaping RCTPromiseResolveBlock,
         rejecter reject: @escaping RCTPromiseRejectBlock
     ) {
-        //         do {
-        // #if canImport(CodePush)
-        //             guard let url = try CodePush.bundleURL() else {
-        //                 reject("GET_CODEPUSH_BUNDLE_URL", "Error getting Codepush Bundle URL", nil)
-        //                 return
-        //             }
-
-        //            // TODO, use path to compute a hash of the assets and add as a 'react_native_bundle_id' key to the resource
-        //             try Embrace.client?.metadata.addResource(key: EmbraceKeys.javaScriptBundleURL.rawValue, value: url.path, .process)
-        //             resolve(true)
-
-        // #else
-        //             resolve(false)
-        // #endif
-        //         } catch let error {
-        //             reject("CHECK_AND_SET_CODEPUSH_BUNDLE_URL", "Error setting Codepush Bundle URL", error)
-        //         }
+                 do {
+         #if canImport(CodePush)
+                     guard let url = try CodePush.bundleURL() else {
+                         reject("GET_CODEPUSH_BUNDLE_URL", "Error getting Codepush Bundle URL", nil)
+                         return
```

Comment:
I wonder if it's better not maintain "codepush" in a specific method, and just let the user set the javascript url if they are using codepush. @jpmunz  what do you think? we can just remove this function and update the doc so we dont care if Codepush changes its name or method in the future

Reply:
I think I'd keep it as is for the purposes of the iOS 6.x migration but I'd like to revisit along with calculating the bundle ID altogether to see if we can reduce code here, noted in https://www.notion.so/embraceio/Rethink-maintaining-support-for-OTA-updates-efe0b4b089ba487d9ab6678b7f7ca491?pvs=4